### ISSUE TITLE
Screen Readers unable to read Status

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#20039]
### Issue type

UI bug

### Bug description

In the View menu, there is an option to show the MuseScore status bar, on by default.

Most screen readers have a hotkey to read any status bar. It does not seem to work for MuseScore. The hotkey varies from screen reader to screen reader.

### Steps to reproduce

1. Ensure the MuseScore status bar is shown (View Menu → Toolbars → Status bar, is checked).
2. Enter some notes on a score.
3. Press the screen reader hotkey to read the status bar, for example:
JAWS: JAWS key + PgDn.
NVDA: NVDA key + End (desktop layout) or NVDA key + Shift + End (laptop layout).
Expected output: the status bar is read.
Actual output: JAWS says nothing, NVDA says no status bar found.

### MuseScore Version

4.1

### Regression

I don't know

### Operating system

Windows + NVDA

**Comment by MarcSabatella** [comment]
In general, what the screen reader reads while navigating your score *is* the status bar, but with some extra info occasionally to make things clearer.  So you're not missing anything.  But indeed, it would be nice if a shortcut designed to read the status bar would re-read the current accessibility info (whether the raw status bar text or the enhanced version).  We do have a built-in command Shift+L (for "get location") that is meant to do that.  But for various reasons, it is hard to force the screen reader to actually read when nothing has changed, so we've relied on  set of tricks that turn out to be unreliable.  So tying into the screen reader built-in method of reading the status bar on command would be great.

**Comment by jmwmusic** [comment]
Most users have been wanting the screen reader to report the status with the aim of getting page/system layout info, similar to in word for page/line.  As this isn't generally on the status I've opened that as a separate issue #33967 and that as the standard status shortcut is most likely to be used automatically that that is used for that purpose.  However, there are still a number of times when being able to get additional info, or as Marc mentioned repeat the info would be useful, so I would now suggest a different shortcut for this purpose that is documented as giving additional info rather than specifically reading the status.
